test/SmokeTests/OOProcSmokeTests/DotNetIsolated/DirectInput.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace DurableNetIsolated.Untyped;
+
+/// <remarks>
+/// The set of functions in this class demonstrate the two different ways input can be resolved for an orchestration.
+/// First is through a parameter. No attribute is needed, we will resolve a parameter which is not otherwise handled and assume
+/// it is the input. This is the preferred way.
+/// Second is through <see cref="TaskOrchestrationContext.GetInput{T}" />. This may be deprecated in the future.
+/// </remarks>
+public static class DirectInput
+{
+    private const string WithInputName = "DirectInputWithInputParameter";
+    private const string NoInputName = "DirectInputNoInputParameter";
+    private const string ActivityName = "DirectInputActivity";
+
+    [Function(nameof(DirectInput))]
+    public static async Task<HttpResponseData> Start(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "post")] HttpRequestData req,
+        [DurableClient] DurableClientContext durableContext,
+        FunctionContext executionContext)
+    {
+        ILogger logger = executionContext.GetLogger(nameof(DirectInput));
+        Input? input = await req.ReadFromJsonAsync<Input>();
```

Comment:
I think this sample would benefit from only using `Input` to represent Orchestrator inputs.
Here, it also represents the input in the HTTP Trigger, which I think is a bit confusing. Can we consider separating these two notions?

Relatedly and even more nitpicky - I would prefer something like `MyInput` over `Input` as the typename here. I think the `My`- prefix helps communicate that there's nothing special about the type `Input` (i.e the user may choose) which I believe is what we're trying to convey here.

Reply:
I changed `Input` to `MyInput`. I did not change the HTTP trigger though, as I believe this is a common scenario to have a payload come in via HTTP and then also serve as the orchestration input. This is just a POCO after all, it can serve many purposes.